pnp/PnpHelper.cs

```diff
@@ -3,7 +3,8 @@
 
 using System.Linq;
 using System.Text;
-using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices;
```

Comment:
Why are we adding a reference to service SDK here? 
If it is to resolve the conflict between `Microsoft.Azure.Devices.Client.Message` and `Microsoft.Azure.Devices.Message`, then it should be resolved by modifying L28 to include the fully qualified class name.

Reply:
The return type here needs to be `Microsoft.Azure.Devices.Client.Message`.